docs/ui/preferences/wsl/integrations.md

```diff
@@ -16,14 +16,18 @@ With WSL, memory and CPU allocation is configured globally across all Linux dist
 [WSL documentation]:
 https://docs.microsoft.com/en-us/windows/wsl/wsl-config#options-for-wslconfig
 
-:::caution warning: breaking change
+### `~/.kube/config`
 
-:::
+Rancher Desktop will create a symlink from `~/.kube/config` inside the distro to the corresponding `%USERPROFILE%\.kube\config` file on the Windows filesystem.
+
+If the `~/.kube/config` file already exist inside the distro and contains only `rancher-desktop` contexts, users, and clusters then it will be deleted and replaced with the symlink.
```

Comment:
If the `~/.kube/config` file already exists inside the distro and contains only `rancher-desktop` contexts, users, and clusters then it will be deleted and replaced with the symlink.

Reply:
I mean the word "exist" to "exists"

Reply:
Amended